src/utils/useHotKey.tsx

```diff
@@ -4,16 +4,39 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
  */
 
-import { useEffect } from 'react';
+import { DependencyList, useEffect } from 'react';
 import Mousetrap from 'mousetrap';
 
-export default (key: string | string[], onKeypress: () => void) => {
+import { sendUsageData } from './usageData';
+
+export interface Shortcut {
+    hotKey: string;
+    title: string;
+    isGlobal: boolean;
+    action: () => void;
+}
+
+export const shortcuts: Shortcut[] = [];
+
+export default (shortcut: Shortcut, deps?: DependencyList) => {
```

Comment:
Exposing the `DependencyList` from `useEffect` is wrong. Just remove the parameter. The way it is used in the RSSI app when calling `useHotKey` is AFAICS not needed at all.

Reply:
But we are re-rendering the same button depending on the `isPaused` bool which is being paused here. Removing this dependencies-paremeter wouldn't give us a way to update the shortcuts-array with a toggle-button since it would need to be two components instead, no?

We added the dependency-array to fix this issue.

Reply:
AFAICS the `isPaused` there does not have any effect. Which effect do you want to achieve? 

If we want to enable users of the hook to update the shortcut then we probably should put `shortcut` in the dependencies of `useEffect`. But that would have some consequences so I first want to understand what you need this for.